data/apply-planet_osm_line.sql

```diff
@@ -32,31 +32,54 @@ UPDATE planet_osm_line
 UPDATE planet_osm_line
   SET mz_label_placement = ST_PointOnSurface(way);
 
-CREATE INDEX planet_osm_line_roads_geom_index ON planet_osm_line USING gist(way) WHERE mz_road_level IS NOT NULL;
-CREATE INDEX planet_osm_line_roads_geom_9_index ON planet_osm_line USING gist(way) WHERE mz_road_level <= 9;
-CREATE INDEX planet_osm_line_roads_geom_12_index ON planet_osm_line USING gist(way) WHERE mz_road_level <= 12;
-CREATE INDEX planet_osm_line_roads_geom_15_index ON planet_osm_line USING gist(way) WHERE mz_road_level <= 15;
-
-CREATE INDEX planet_osm_line_transit_geom_index ON planet_osm_line USING gist(way) WHERE mz_transit_level IS NOT NULL;
-CREATE INDEX planet_osm_line_transit_geom_6_index ON planet_osm_line USING gist(way) WHERE mz_transit_level <= 6;
-CREATE INDEX planet_osm_line_transit_geom_9_index ON planet_osm_line USING gist(way) WHERE mz_transit_level <= 9;
-CREATE INDEX planet_osm_line_transit_geom_12_index ON planet_osm_line USING gist(way) WHERE mz_transit_level <= 12;
-CREATE INDEX planet_osm_line_transit_geom_15_index ON planet_osm_line USING gist(way) WHERE mz_transit_level <= 15;
-
-CREATE INDEX planet_osm_line_earth_geom_index ON planet_osm_line USING gist(way) WHERE mz_earth_min_zoom IS NOT NULL;
-CREATE INDEX planet_osm_line_earth_geom_9_index ON planet_osm_line USING gist(way) WHERE mz_earth_min_zoom <= 9;
-CREATE INDEX planet_osm_line_earth_geom_12_index ON planet_osm_line USING gist(way) WHERE mz_earth_min_zoom <= 12;
-CREATE INDEX planet_osm_line_earth_geom_15_index ON planet_osm_line USING gist(way) WHERE mz_earth_min_zoom <= 15;
-
-CREATE INDEX planet_osm_line_water_geom_index ON planet_osm_line USING gist(way) WHERE mz_water_min_zoom IS NOT NULL;
-CREATE INDEX planet_osm_line_water_geom_9_index ON planet_osm_line USING gist(way) WHERE mz_water_min_zoom <= 9;
-CREATE INDEX planet_osm_line_water_geom_12_index ON planet_osm_line USING gist(way) WHERE mz_water_min_zoom <= 12;
-CREATE INDEX planet_osm_line_water_geom_15_index ON planet_osm_line USING gist(way) WHERE mz_water_min_zoom <= 15;
-
-CREATE INDEX planet_osm_line_boundary_geom_index ON planet_osm_line USING gist(way) WHERE mz_boundary_min_zoom IS NOT NULL;
-CREATE INDEX planet_osm_line_boundary_geom_9_index ON planet_osm_line USING gist(way) WHERE mz_boundary_min_zoom <= 9;
-CREATE INDEX planet_osm_line_boundary_geom_12_index ON planet_osm_line USING gist(way) WHERE mz_boundary_min_zoom <= 12;
-CREATE INDEX planet_osm_line_boundary_geom_15_index ON planet_osm_line USING gist(way) WHERE mz_boundary_min_zoom <= 15;
+-- only these 2 columns are relevant in lower zoom queries
```

Comment:
Yea, I thought about making some more targeted changes, but since we know that the previous indexes aren't going to be relevant any more, I figured it would be cleaner just to remove all the old ones and make new ones that would be relevant. This file is responsible for setting up new databases, and there's a migration that will remove the old indexes and create new ones.